#include "game_player_scene_replied_handler.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
void GamePlayerSceneServiceRepliedHandler::EnterScene(entt::entity player,const ::GsEnterSceneRequest* request,
	     google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

void GamePlayerSceneServiceRepliedHandler::LeaveScene(entt::entity player,const ::GsLeaveSceneRequest* request,
	     google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

void GamePlayerSceneServiceRepliedHandler::EnterSceneS2C(entt::entity player,const ::EnterSceneS2CRequest* request,
	     EnterScenerS2CResponse* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

